huxley/core/migrations/0003_auto_20150618_1956.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
You can almost certainly squash this migration into the previous one. Check out the migrations documentation on how to do that.